lte/gateway/python/magma/pipelined/tests/pipelined_test_util.py

```diff
@@ -552,3 +553,26 @@ def get_ovsdb_port_tag(port_name: str) -> str:
             pass
 
 
+def get_ovsdb_port_tag(port_name: str) -> str:
+    dump1 = subprocess.Popen(["ovsdb-client", "dump", "Port", "name", "tag"],
+                             stdout=subprocess.PIPE)
+    for port in dump1.stdout.readlines():
+        if port_name not in str(port):
+            continue
+        try:
+            tokens = str(port).strip().split()
+            port_name = tokens[0]
+            tag = tokens[1]
+            if port_name == port_name:
+                return str(tag).split('\\')[0]
+        except ValueError:
+            pass
+
+
+def get_iface_ipv4(iface: str):
```

Comment:
Maybe annotate that it returns a list(I'm assuming its string)?
`-> List[string]`